<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupInfo</class>
 <widget class="QWidget" name="GroupInfo">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Group information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Segoe UI Variable Small Semibol</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Group info</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="groupName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Group tag:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="groupTag">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Group owner:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Show owner</string>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/messagemenu/ui/images/contactsPH.ico</normaloff>:/messagemenu/ui/images/contactsPH.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Creation date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="groupDate">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
